### PR TITLE
Make main thread actions sequential

### PR DESCRIPTION
**Pull request by Pablito2020**
As listed in issue: #16, the pomodoro main thread can handle more operations sequentially without the overhead of creating tokio threads.

It should be noted that the CREATE option has  been refactored and moved to a separate method, but it would still be quite necessary a refactor for the whole system (isolating different logics, like for example the DataBase logic has to be completely decoupled from main.rs, the parsing arguments part should be delegated to argument.rs, and so much more) but this has to be thinked and talked in another issue.

In addition, I can say that theoretically this pull request does not break any of the current functionalities of the code (since I have tested the application and it seems to work the same as before), although, as at the moment there are no unit tests, I cannot ensure that this is true for the 100% of cases.

### CODE REVIEW

**Comment by 24seconds** [comment]
[Request]
It seems already merged commits exist, ba6a416 and 
080a0b7. Could you make commit logs clear?

**Comment by 24seconds** [comment]
[Request]
Could you run this command right under the directory and resolve warning? About `Hint` warning, you can ignore it
```
cargo clippy
```

File: src/main.rs [hunk 1]
```diff
@@ -51,167 +45,106 @@ async fn main() -> Result<(), Box<dyn Error>> {
...
-                let _ = oneshot_tx.send(String::from("Query succeed"));
-            }
-            (TEST, Some(_)) => {
-                debug!("Message:NotificationTest called!");
-                notify_work(&configuration.clone()).await?;
-                debug!("Message:NotificationTest done");
+                db::delete_all_notification(glue.clone()).await;
 
-                let _ = oneshot_tx.send(String::from("NotificationTest called"));
+                debug!("Message::DeleteAll done");
```
Comment (24seconds) [inline src/main.rs:125]: [Request]
Here we previously make output message, `All notifications deleted` but it seems it's omitted. Could you add this?

**Review by 24seconds — CHANGES_REQUESTED** [review]
Thank you for the work.
Looks good overall

**Comment by Pablito2020** [comment]
The changes has been made to the branch. I had to do a "git rebase" for deleting the documentation commit and that's why in the pull request now appears: 

_Pablito2020 force-pushed the main-sequential branch from e9c2030 to af68323 1 minute ago_

File: src/main.rs [hunk 2]
```diff
@@ -210,6 +203,9 @@ async fn create_notification<'a>(
 
     if work_time == 0 && break_time == 0 {
         eprintln!("work_time and break_time both can not be zero both");
+        // TODO: This shouldn't return Ok, since it is an error, but for now,
+        // is just a "temporal fix" for returning from the function.
```
Comment (24seconds) [inline src/main.rs:207]: [comment]
Good

**Review by 24seconds — APPROVED** [review]
Thank you for addressing feedback! LGTM

**Comment by 24seconds** [comment]
[Question]
I don't know whether you can merge yourself or not, if you can, please do it!

**Comment by Pablito2020** [comment]
> [Question] I don't know whether you can merge yourself or not, if you can, please do it!
I can't merge it to the repository because I don't have write access to it.

**Comment by 24seconds** [comment]
Aha! Sorry, I didn't know that. Thank you for the check!

**Pull request merged**